mintpy/ifgram_inversion.py

```diff
@@ -136,66 +169,35 @@ def cmd_line_parse(iargs=None):
 
     # check input file type
     atr = readfile.read_attribute(inps.ifgramStackFile)
-    if atr['FILE_TYPE'] not in ['ifgramStack']:
-        raise ValueError('input is {} file, support ifgramStack file only.'.format(atr['FILE_TYPE']))
+    assert atr['FILE_TYPE'] == 'ifgramStack', 'input is {} file, only support ifgramStack file.'.format(k)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/insarlab/MintPy/pullRequest?prid=5671549)